staging/src/k8s.io/api/testdata/HEAD/resource.k8s.io.v1alpha3.DeviceClass.json

```diff
@@ -67,30 +67,6 @@
           }
         }
       }
-    ],
```

Comment:
Shouldn't this still be part of v1alpha3? It's a little confusing.

Reply:
Ditto on all the files below. I guess since we are removing fields from v1alpha3, they are just silently ignored/dropped in existing manifests? Seems very strange to be changing versioned test data.

Reply:
That's just how this test data works. That this PR needs to modify the test data shows that it is a breaking API change.

Reply:
You wrote:
> except if feels "wrong" to be changing the v1alpha3 API, rather than doing a v1alpha4 with the removed fields, etc. Are we sure that's the standard way to do it?

That is odd thing about API versioning in Kubernetes: when adding or (in this case) removing fields the round-tripping rules require that *all* supported API versions need to be modified in lockstep. Forget about "v1alpha3" describing a fixed, immutable set of fields - that's simply not how the Kubernetes API works. Yes, it is confusing.